docs/architecture.md

```diff
@@ -1,240 +1,44 @@
-
-#### Monitoring and Auditing
-
-* Every execution is captured.
-* Every task run has captured input and outputs.
-* CloudWatch Metrics can be [used for monitoring](https://docs.aws.amazon.com/step-functions/latest/dg/procedure-cw-metrics.html) many of the events with the StepFunctions. It can also generate alarms for the whole process.
-* Visual report of the entire configuration.
-  * Errors and success states are highlighted visually in the flow.
-
-#### Data Provenance
-
-* Monitoring and auditing ensures we know the data that was given to a task.
-* Workflows are versioned and the state machines stored in AWS Step Functions are immutable. Once created they cannot change.
-* Versioning of data in S3 or using immutable records in S3 will mean we always know what data was created as the result of a step or fed into a step.
-
-## Appendix
-
-### Open Questions
-
-* How does each step know which configuration element to use?
-  * The configuration for an individual step is located in the configuration details passed in through the message. The executer needs to know which step it is so it can retrieve the correct details from the full config. The problem is unless it is hard coded in the step it does not know what it's name is. This makes it difficult to reuse a step in multiple locations.
-* How do we version the workflows?
-
-### Schemas
-
-#### Message JSON Schema
-
-The Message JSON schema defines the structure of the message sent to and returned from tasks.
+<img src="/images/cumulus-arch-diagram.png">
 
-<a href="/schemas/message_schema.json" target="_blank">message_schema.json</a>
+## Cloudformation Stacks
 
-<a href="/schemas/example-data/example-message-envelope.json" target="_blank">Example Message Message</a>
+1. **IAM:** Sets up IAM roles for AWS services deployed by the Cumulus stack to be assigned the required policies.
+2. **Cumulus:**
+    * Lambda functions and ECS activities (required for workflows)
+    * Step functions (aka workflows)
+    * Elasticsearch for search on workflow executions and logs
+    * DynamoDB for storing (earthdata) users, rules, collections and providers.
+3. **Cumulus API Default** and **Cumulus API V1:** Nested stacks deploying API Gateway resources
 
-### Example GIBS Ingest Architecture
 
-This shows the GIBS Ingest Architecture as an example of the use of the Ingest Workflow Architecture.
+## Cumulus Dashboard
 
-* The GIBS Ingest Architecture consists of two workflows per collection type. There is one for discovery and one for ingest. The final stage of discovery triggers multiple ingest workflows for each MRF granule that needs to be generated.
-* It demonstrates both lambdas as tasks and a container used for MRF generation.
+Uses S3 static website hosting built using the [cumulus-dashboard](https://github.com/cumulus-nasa/cumulus-dashboard) repository.
 
-#### GIBS Ingest Workflows
+## API Lambda Tasks
```

Comment:
What is the reasoning behind documenting this? This seems kind of behind the scenes and something that wouldn't necessarily be relevant to an end user. Will defer to Patrick here.

Reply:
Ya we can remove this - was really for my own knowledge development. I thought it was helpful to annotate the architecture diagram but these tasks don't clearly map from that diagram.